hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configuration.java

```diff
@@ -106,12 +106,6 @@
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
 
-import com.google.common.base.Preconditions;
-import com.google.common.base.Strings;
-
-import static org.apache.commons.lang3.StringUtils.isBlank;
-import static org.apache.commons.lang3.StringUtils.isNotBlank;
-
```

Comment:
your IDE is moving imports around which is shouldn't; this always makes backporting of patches to a key class harder. Can you leave out all changes to the imports except for those classes added/removed

Reply:
Is there an import order rule? I checked several files but can not find the pattern, static import can be anywhere, and no alphabetical order...